public/whiteboard/js/main.js

```diff
@@ -36,17 +34,21 @@ window.p5Obj = new p5((p) => {
   // }
 
   p.preload = () => {
-    img = p.loadImage('https://img.icons8.com/emoji/452/-emoji-sparkles.png')
-    // Currently have a dummy image but we will be recieving images from the backend
-    // img = loadImage('https://media.istockphoto.com/vectors/cute-cartoon-cockatiel-couple-vector-id1160619330?b=1&k=6&m=1160619330&s=612x612&w=0&h=V6NM2xCwQ16AyIEK0prZKqm6NNjMfgZN7VUyL6HaELs=')
+    // Currently have a dummy id and image but we will be receiving this from the backend
+    user = {
```

Comment:
This should probably be stored in a user class or something once we flesh out the different things we need, like switching brushes